src/Test.FunctionalTests.Sarif/Test.FunctionalTests.Sarif.csproj

```diff
@@ -47,7 +47,7 @@
 
   <ItemGroup>
     <PackageReference Include="FluentAssertions" Version="5.10.2" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.0-preview-20220707-01" />
```

Comment:
>"17.4.0-preview-20220707-01" [](http://example.com/codeflow?start=63&length=28)

I see we are using preview version, looks like we need a certain fix in that version.
can you add a comment in the PR for information purpose.

Reply:
This preview version includes the update to Newtonsoft 13.0.1